htlcswitch/link.go

```diff
@@ -1418,6 +1418,12 @@ func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
 
 		l.cfg.Peer.SendMessage(false, htlc)
 
+		// Send a forward event notification to htlcNotifier.
+		info, pid := downstreamPacketInfo(pkt)
```

Comment:
Do we indeed want to expose the pid? Or do we stick to reporting the outgoing chan/htlc id and let users of the notifier associated htlcs and payments using the to-be-added circuit key in the payment db?

Reply:
Indeed, cut the pid and payment hash 👍